### PR TITLE
feat: Cloud Adapter for streaming tests

### PR DESCRIPTION
**Pull request by joshgummersall**
- Moves streaming e2e tree into the `testing` directory
- Code formatting
- Replaces `BotFrameworkAdapter` with `CloudAdapter`

#minor

### CODE REVIEW

**Comment by joshgummersall** [comment]
Note: once this lands I'll reconfigure the ADO pipeline(s) to point to the new location.

**Review by mdrichardson — APPROVED** [review]

File: testing/streaming-e2e/bot/index.js
Comment (mdrichardson) [inline testing/streaming-e2e/bot/index.js:88]: Should this be `async`, or does it not matter?

Comment (joshgummersall) [reply testing/streaming-e2e/bot/index.js:88]: [Not in this case 😄](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions#function_body). The return value is propagated because there are no braces.

File: testing/streaming-e2e/bot/index.js
Comment (mdrichardson) [inline testing/streaming-e2e/bot/index.js:135]: Maybe either `DIRECT_LINE_SECRET?.substr(-3)` or do an argument check (if not already done elsewhere)?

**Review by mdrichardson — APPROVED** [review]
Approved, pending the one unresolved comment I left.

File: testing/streaming-e2e/bot/index.js
Comment (joshgummersall) [reply testing/streaming-e2e/bot/index.js:135]: I wanted to keep this change scoped to just Cloud Adapter. As this is code for CI, I think it's okay to expect the environment is set up correctly.

**Pull request merged**
